driver/opentelekomcloud.go

```diff
@@ -266,6 +268,23 @@ func (d *Driver) Authenticate() error {
 	return d.client.Authenticate(opts)
 }
 
+func (d *Driver) createFloatingIP() error {
```

Comment:
mb rename to `createAndBind` ?

Reply:
It's named like others do, `createVPC`, `createSubnet`, etc. name `createAndBindFloatingIP` would be too long